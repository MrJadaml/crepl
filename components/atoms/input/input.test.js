import React from 'react';
import { Input } from './index';
import { cleanup, render } from '@testing-library/react';

afterEach(cleanup);

describe('Input', () => {
  it('should render an input', () => {
    const { asFragment } = render(<Input />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should accept types', () => {
    const { asFragment } = render(<Input type="checkbox" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
