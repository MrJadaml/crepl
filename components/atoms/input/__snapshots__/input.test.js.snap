// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should accept types 1`] = `
<DocumentFragment>
  <input
    class="input"
    type="checkbox"
  />
</DocumentFragment>
`;

exports[`Input should render an input 1`] = `
<DocumentFragment>
  <input
    class="input"
    type="text"
  />
</DocumentFragment>
`;
